New log file opened at Tue May 12 11:38:12 CEST 2015 by 98996@Laban
Taking next connection to: 49424
New client attempting to connect on port: 49425-49424
  Address = /130.237.156.217:49424
This client on port 49425 has uuid 52921316-71cf-4f17-ae1c-d9baa0c2fcab configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-52921316-71cf-4f17-ae1c-d9baa0c2fcab
Taking next connection to: 49424
New client attempting to connect on port: 49426-49424
  Address = /130.237.156.217:49424
Handshake exception on socket: 49426
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49424
New client attempting to connect on port: 49427-49424
  Address = /130.237.156.217:49424
This client on port 49427 has uuid cb07d1dd-7380-46de-8892-f463a31961b4 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-52921316-71cf-4f17-ae1c-d9baa0c2fcab, activator-cb07d1dd-7380-46de-8892-f463a31961b4
Taking next connection to: 49424
New client attempting to connect on port: 49428-49424
  Address = /130.237.156.217:49424
Handshake exception on socket: 49428
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49424
New client attempting to connect on port: 49429-49424
  Address = /130.237.156.217:49424
This client on port 49429 has uuid ab4ded36-71b4-4dfc-b2d2-02bbf7a57b73 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-cb07d1dd-7380-46de-8892-f463a31961b4, play-fork-run-ab4ded36-71b4-4dfc-b2d2-02bbf7a57b73
Taking next connection to: 49424
New client attempting to connect on port: 49536-49424
  Address = /130.237.156.217:49424
Handshake exception on socket: 49536
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49424
New client attempting to connect on port: 49537-49424
  Address = /130.237.156.217:49424
This client on port 49537 has uuid 878fdd30-71d0-46fa-8507-e86cee162a78 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-cb07d1dd-7380-46de-8892-f463a31961b4, play-fork-run-878fdd30-71d0-46fa-8507-e86cee162a78
Taking next connection to: 49424
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49424
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49424
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49424
